diff --git a/eslint.config.js b/eslint.config.js
index a46ca1503637fdfc0b3b637638a041558aa52d68..98d24aab216fb91699365882f666a9c1fe345adc 100644
--- a/eslint.config.js
+++ b/eslint.config.js
@@ -5,7 +5,7 @@ import eslintPluginReact from 'eslint-plugin-react';
 import eslintPluginReactCompiler from 'eslint-plugin-react-compiler';
 import eslintPluginReactHooks from 'eslint-plugin-react-hooks';
 
-const [reactHooksFlatRecommended] = eslintPluginReactHooks.configs['flat/recommended'];
+const reactHooksFlatRecommended = eslintPluginReactHooks.configs.flat.recommended;
 
 export default [
   ...jsConfig,
